internal/cloudapi/v2/server.go

```diff
@@ -37,6 +37,9 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/worker/clienterrors"
 )
 
+// How long to wait for a depsolve job to finish
+const depsolveTimeout = 5
```

Comment:
The tiniest nitpick:
Should mention this is in minutes since it's now a few hundred lines away from its usage.  Can be part of the comment
```
How long to wait for a depsolve to finish (in minutes)
```
or the variable name
```
depsolveTimeoutMins = 5
```